core/src/main/java/io/questdb/cairo/sql/async/PageFrameReduceTask.java

```diff
@@ -158,25 +160,33 @@ public boolean isOutOfMemory() {
 
     public void of(PageFrameSequence<?> frameSequence, int frameIndex) {
         this.frameSequence = frameSequence;
-        this.frameMemoryPool.of(frameSequence.getPageFrameAddressCache());
         this.frameSequenceId = frameSequence.getId();
         this.type = frameSequence.getTaskType();
```

Comment:
nit: could we maintain consistent naming convention. `type` is too generic. Should be `taskType`

Reply:
Done in 5d2a3f2f48268939f5793df51bfbe26356c3179b